tests/PySys/call_tedge_config_help/run.py

```diff
@@ -0,0 +1,31 @@
+import pysys
+from pysys.basetest import BaseTest
+
+import time
+
+"""
+Validate command line option help
+
+Given a running system
+When we call tedge help
+Then we find the string USAGE: in the output
+Then we find the string FLAGS: in the output
+Then we find the string SUBCOMMANDS: in the output
+"""
+
+
+class PySysTest(BaseTest):
+    def execute(self):
+        tedge = "/usr/bin/tedge"
+
+        proc = self.startProcess(
+            command=tedge,
+            arguments=["config", "help"],
+            stdouterr="tedge",
+            expectedExitStatus="==0",
+        )
+
+    def validate(self):
+        self.assertGrep("tedge.out", "USAGE:", contains=True)
+        self.assertGrep("tedge.out", "FLAGS:", contains=True)
+        self.assertGrep("tedge.out", "SUBCOMMANDS:", contains=True)
```

Comment:
That validation is a bit too terse. For any tedge subcommand, this template that's being validated would be the same. Shouldn't we validate more? Like the actual subcommands for `config` in this case which are `get`, `set`, `list` etc. SImilarly we'd want to validate the exact subcommands list for other tedge commands as well.

Reply:
Let's discuss that. Maybe it's not that many checks but we make sure, that there is a help message and we do not need to update the test when a description is changed (when we have a test for that somewhere else).